src/media/js/views/feedback.js

```diff
@@ -27,45 +27,42 @@ define('views/feedback',
                 message: gettext('There was a problem submitting your feedback. Try again soon.')
             });
         });
-    });
+    }));
 
-    // Init desktop feedback form modal trigger.
-    function addFeedbackModal() {
+    function addFeedbackModalDesktop() {
+        // Desktop feedback modal.
         if (!caps.widescreen() ||
             urls.reverse('feedback') === window.location.pathname) {
             return;
         }
-        if (!$('.main.feedback:not(.modal)').length && !$('.feedback.modal').length) {
-            z.page.append(nunjucks.env.render('settings/feedback.html'));
+        if (!$('.main.feedback').length && !$('mkt-prompt.feedback').length) {
+            z.page.append(nunjucks.env.render('_includes/feedback_modal.html'));
         }
-        z.body.trigger('decloak');
```

Comment:
I thought this toggles the "overflowed" class which basically sets a `body {overflow: hidden}` for the modal background mask ("cloak").